src/librustdoc/html/static/js/search.js

```diff
@@ -1996,14 +2006,17 @@ function initSearch(rawSearchIndex) {
             const itemTypes = crateCorpus.t;
             // an array of (String) item names
             const itemNames = crateCorpus.n;
-            // an array of (String) full paths (or empty string for previous path)
-            const itemPaths = crateCorpus.q;
+            // an array of [(Number) item index,
+            //              (String) full path]
```

Comment:
This comment drops the description of the "previous path" semantic. I think the new semantic is "Items whose item index is not in this array get their full path from the next-lowest item index that _does_ occur in this array."